packages/now-cli/test/dev/fixtures/test-zero-config-rewrite/.gitignore

```diff
@@ -0,0 +1 @@
+.now
```

Comment:
This one seems redundant wrt the root `.gitignore` file change.

Reply:
Yep, I removed it